build/charts/antrea/README.md

```diff
@@ -72,6 +72,7 @@ Kubernetes: `>= 1.16.0-0`
 | controller.tolerations | list | `[{"key":"CriticalAddonsOnly","operator":"Exists"},{"effect":"NoSchedule","key":"node-role.kubernetes.io/master"},{"effect":"NoSchedule","key":"node-role.kubernetes.io/control-plane"}]` | Tolerations for the antrea-controller Pod. |
 | defaultMTU | int | `0` | Default MTU to use for the host gateway interface and the network interface of each Pod. By default, antrea-agent will discover the MTU of the Node's primary interface and adjust it to accommodate for tunnel encapsulation overhead if applicable. |
 | disableTXChecksumOffload | bool | `false` | Disable TX checksum offloading for container network interfaces. It's supposed to be set to true when the datapath doesn't support TX checksum offloading, which causes packets to be dropped due to bad checksum. It affects Pods running on Linux Nodes only. |
+| dnsInterceptionRateLimit | int | `500` | DNSInterceptionRateLimit is the rate limiting of DNS interception introduced by FQDN. It stands for the rate as packets per second(pps) and the burst/queue size will be automatically set to 2 times of it. |
```

Comment:
Could we make it more general to support configuring other rate-limiting as https://github.com/antrea-io/antrea/issues/5358 asks?

Reply:
Sure